.github/workflows/deploy.yml

```
@@ -0,0 +1,37 @@
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.VERCEL_TOKEN }}
+        with:
+          source-directory: 'output'
+          destination-github-username: sohyun127
+          destination-repository-name: DanJi-FE
+          user-email: ${{ secrets.VERCEL_USER_EMAIL }}
+          commit-message: ${{ github.event.commits[0].message }}
+          target-branch: main
+
+      - name: Test get variable exported by push-to-another-repository
+        run: echo $DESTINATION_CLONED_DIRECTORY
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)